events/2020-10-10.interspeciesio-ml-workshop.md

```diff
@@ -1,25 +1,109 @@
+*   [Deep Learning Based Phase Reconstruction for Speaker Separation: A Trigonometric Perspective](https://arxiv.org/abs/1811.09010)
+*   [Divide and Conquer: A Deep CASA Approach to Talker-independent Monaural Speaker Separation](https://arxiv.org/abs/1904.11148)
+*   [Dual-Path RNN: Efficient Long Sequence Modeling for Time-Domain Single-Channel Speech Separation](https://arxiv.org/abs/1910.06379)
+*   [A Comprehensive Study of Speech Separation: Spectrogram vs Waveform Separation](https://arxiv.org/abs/1905.07497)
+*   [FurcaNeXt: End-to-End Monaural Speech Separation with Dynamic Gated Dilated Temporal Convolutional Networks](https://arxiv.org/abs/1902.04891)
+*   [Improving Universal Sound Separation Using Sound Classification](https://arxiv.org/abs/1911.07951)
+*   [SpEx: Multi-Scale Time Domain Speaker Extraction Network](https://arxiv.org/abs/2004.08326)
+*   [Meta-Learning Extractors for Music Source Separation](https://arxiv.org/abs/2002.07016)
+*   [Unsupervised Learning of Semantic Audio Representations](https://arxiv.org/abs/1711.02209)
+*   [Sudo rm -rf: Efficient Networks for Universal Audio Source Separation](https://arxiv.org/abs/2007.06833)
+*   [Unsupervised Sound Separation Using Mixtures of Mixtures](https://arxiv.org/abs/2006.12701)
+*   [Listen to What You Want: Neural Network-based Universal Sound Selector](https://arxiv.org/abs/2006.05712)
+*   [Speech Separation Based on Multi-Stage Elaborated Dual-Path Deep BiLSTM with Auxiliary Identity Loss](https://arxiv.org/abs/2008.03149)
+*   [Identify Speakers in Cocktail Parties with End-to-End Attention](https://arxiv.org/abs/2005.11408)
 
-## Next meeting
+# 
+**Machine Learning Concepts**
 
-- **TODO:** Do we want to set an approximate time for next meeting?
\ No newline at end of file
+*   Matched Filters
+*   Hilbert Huang Transforms
+*   Lombard Effect
+*   Dip-Listening Hypothesis
+*   Noise-Invariant, Rule-Encoding, and Temporal Edge Neurons
+*   FastICA (Independent Component Analysis), Point Source Separation (PSS), Non-Negative Matrix Factorization (NMF)
+*   Cortical Representation of Speech, Auditory Object Analysis
+*   Deep Scattering Spectrum, Kymatio, Invariant Scattering Convolution Networks
+*   Error-Gated Hebbian Rule
+*   Probabilistic Latent Component Analysis (PLCA), Robust PCA (RPCA), Low-Rank Modeling
+*   Progressive Learning
+*   Denoising Autoencoders
+*   Temporal Convolutional Networks (TCNs)
+*   Perfect Reconstruction Filterbank (PRFB), Gammatone Filterbank
+*   Deep Feature Losses, Short-Time Objective Intelligibility (STOI), Perceptual Evaluation of Speech Quality (PESQ)
+*   Wiener Filtering
+*   Permutation Invariant Training
+*   Supervised Independent Vector Analysis
+*   Phase Spectra and Group Delay
+*   Spectrogram Fusion
+*   Self-Organizing Background Subtraction
+*   Phase-Aware Signal Processing
+*   Deep Clustering, Deep Attractor, and Selective Hearing Networks
+*   Multimodal/Crossmodal Fusion
```

Comment:
These are not all deep learning concepts ... are these things that @pcbermant mentioned that we're just including as a reference?